### ISSUE TITLE
"podman machine start" crashes M1 Macbook Pro

### ISSUE THREAD

**Issue by JostLuebbe** [containers/podman#14652]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

Running `podman machine start' crashes M1 Macbook Pro

**Steps to reproduce the issue:**

1. brew install podman

2. podman machine init

3. podman machine start

**Describe the results you received:**

Macbook crashes and restarts.

**Describe the results you expected:**

Podman machine starts.

**Additional information you deem important (e.g. issue happens only occasionally):**

I tried downgrading my qemu version to 6.2 from 7.0 since that seemed to solve some other people's issues with starting a machine, but the problem persisted. 

**Output of `podman version`:**

```
$ podman --version
podman version 4.1.1
```

**Output of `podman info --debug`:**

```
$ podman info --debug
Cannot connect to Podman. Please verify your connection to the Linux system using `podman system connection list`, or try `podman machine init` and `podman machine start` to manage a new Linux VM
Error: unable to connect to Podman. failed to create sshClient: connection to bastion host (ssh://core@localhost:49757/run/user/503/podman/podman.sock) failed: dial tcp [::1]:49757: connect: connection refused
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
$ brew info podman
podman: stable 4.1.1 (bottled), HEAD
Tool for managing OCI containers and pods
https://podman.io/
/opt/homebrew/Cellar/podman/4.1.1 (174 files, 46.4MB) *
  Poured from bottle on 2022-06-17 at 11:12:35
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/podman.rb
License: Apache-2.0
==> Dependencies
Build: go ✘, go-md2man ✘
Required: qemu ✔
==> Options
--HEAD
	Install HEAD version
==> Caveats
zsh completions have been installed to:
  /opt/homebrew/share/zsh/site-functions
==> Analytics
install: 18,206 (30 days), 59,671 (90 days), 157,270 (365 days)
install-on-request: 18,077 (30 days), 59,510 (90 days), 157,087 (365 days)
build-error: 1 (30 days)
```

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide? (https://github.com/containers/podman/blob/main/troubleshooting.md)**

Yes

**Additional environment details (AWS, VirtualBox, physical, etc.):**

```
$ uname -a
Darwin LYRQXXYXCWN 21.4.0 Darwin Kernel Version 21.4.0: Fri Mar 18 00:46:32 PDT 2022; root:xnu-8020.101.4~15/RELEASE_ARM64_T6000 arm64
```

```
Software:

    System Software Overview:

      System Version: macOS 12.3.1 (21E258)
      Kernel Version: Darwin 21.4.0

Hardware:

    Hardware Overview:

      Model Name: MacBook Pro
      Model Identifier: MacBookPro18,1
      Chip: Apple M1 Pro
      Total Number of Cores: 10 (8 performance and 2 efficiency)
      Memory: 16 GB
      System Firmware Version: 7459.121.3
      OS Loader Version: 7459.101.3
```

**Comment by rhatdan** [comment]
Can  you upgrade your mac to 12.4.  We have heard that this fixes the issue.

**Comment by JostLuebbe** [comment]
> Can you upgrade your mac to 12.4. We have heard that this fixes the issue.

Hi @rhatdan, thanks for the quick reply! I'll ask the system administrator if that's possible and get back to you.

**Comment by rhatdan** [comment]
Since this issue is somewhat out of our control and a discussion is happening in #14563, I am closing this issue.

**Comment by adrian-moisa** [comment]
Apparently I'm facing the exact same issue. I'm currently on 12.0. Currently available version is 13.3.1.
